src/corelibs/U2Formats/src/FastaFormat.cpp

```diff
@@ -253,40 +241,40 @@ static void load(IOAdapter *io, const U2DbiRef &dbiRef, const QVariantMap &fs, Q
                 os.setError(FastaFormat::tr("File \"%1\" contains too many sequences to be displayed. "
                                             "However, you can process these data using instruments from the menu <i>Tools -> NGS data analysis</i> "
                                             "or pipelines built with Workflow Designer.")
-                                .arg(io->getURL().getURLString()));
+                                .arg(reader.getURL().getURLString()));
                 break;
             }
             memoryLocker.tryAcquire(800);
             CHECK_OP_BREAK(os);
             U2Sequence seq = seqImporter.finalizeSequenceAndValidate(os);
-            if (os.hasError() && os.getError() == U2SequenceImporter::EMPTY_SEQUENCE_ERROR) {
+            if (os.getError() == U2SequenceImporter::EMPTY_SEQUENCE_ERROR) {
                 os.setError("");
-                emptySeqNames << headerLine;
+                emptySeqNames << headerLine.mid(1);
```

Comment:
Add a comment, that `mid` removes `>` from name, it's not pretty obvious

Reply:
Reworked the code to avoid repeated .mid(1) calls